ui/core.js

```diff
@@ -88,7 +88,9 @@ $.extend( $.ui, {
 		if ( element && element.nodeName.toLowerCase() !== "body" ) {
 			$( element ).blur();
 		}
-	}
+	},
+
+	escapeId: new RegExp( /([!"#$%&'()*+,./:;<=>?@[\]^`{|}~])/g )
```

Comment:
probably a better solution since you still need to know the other half of the escape